src/main/java/by/itech/lab/supplier/controller/UserController.java

```diff
@@ -1,29 +1,83 @@
 package by.itech.lab.supplier.controller;
 
-import by.itech.lab.supplier.auth.UserImpl;
-import by.itech.lab.supplier.dto.CustomerDto;
-import lombok.AllArgsConstructor;
+import by.itech.lab.supplier.constant.ApiConstants;
+import by.itech.lab.supplier.domain.User;
+import by.itech.lab.supplier.dto.UserDto;
+import by.itech.lab.supplier.service.UserService;
+import by.itech.lab.supplier.service.validator.IdValidator;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.http.HttpHeaders;
 import org.springframework.http.HttpStatus;
-import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.DeleteMapping;
 import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.ResponseStatus;
 import org.springframework.web.bind.annotation.RestController;
 
+import javax.validation.Valid;
+import java.net.URI;
+import java.net.URISyntaxException;
 import java.util.List;
 
-import static by.itech.lab.supplier.constant.ApiConstants.URL_COMPANIES;
-import static by.itech.lab.supplier.constant.ApiConstants.URL_USER;
-
+@CrossOrigin
```

Comment:
Ive thought that its configured for a whole application in config class, is there really needed that annotation?